nilearn/decoding/space_net_solvers.py

```diff
@@ -140,12 +141,11 @@ def _graph_net_adjoint_data_function(X, w, adjoint_mask, grad_weight):
         Unmasked, ravelized weights map.
 
     grad_weight: float
-        l1_ratio * alpha
+        l1_ratio * alpha.
 
     Returns
     -------
-    float
-
+    ndarray
```

Comment:
I think that the comment @bthirion made is precisely to have more that the type here. We want to know exactly the semantics of the returned value.